### DISCUSSION TITLE
How to remove particular activity while deploying to other environment.

### DISCUSSION THREAD

**Discussion by namanpatel12** [Azure-Player/azure.datafactory.tools#296] [Q&A]
Hi @NowinskiK ,

I do need to delete particular activity while deploying to next stage. I have added following in config file but does not work. Can you please help? (Not just exclude from the deployment, but just simply delete/remove activity from pipeline)
[Pipeline.txt](https://github.com/Azure-Player/azure.datafactory.tools/files/11108341/Pipeline.txt)

pipeline,PL_EX_GLS_Master_FSH,-activities["Data masking"],

Regards,
Naman

**Comment by NowinskiK** [comment]
What's the current outcome? Error? Nothing?

**Comment by namanpatel12** [comment]
Hi @NowinskiK ,

Thanks for the response.

Release was successful with as it is. That activity was there. Tried so many different options to change query of activity as it was not getting rid of but no luck so far. Syntax error in each of them.

pipeline,PL_EX_GLS_Master_FSH,-activities["Data Masking"],
pipeline,PL_EX_GLS_Master_FSH,activities["Data Masking"].typeProperties.scripts.text,"select 1"
pipeline,PL_EX_GLS_Master_FSH,activities["Data Masking"].typeProperties.scripts,"{""type"": ""Query"",""text"":""select 1""}"
pipeline,PL_EX_GLS_Master_FSH,activities["Data Masking"].typeProperties.scripts,"[{""type"": ""Query"",""text"":""select 1""}]"
pipeline,PL_EX_GLS_Master_FSH,activities["Data Masking"].typeProperties.scripts,"{""type"": ""Query"",""text"": ""select 1""}"

![image](https://user-images.githubusercontent.com/84165941/229016649-94aeee97-6ce8-4ab4-bbec-4378bc75da99.png)
![image](https://user-images.githubusercontent.com/84165941/229016676-e89936fc-e057-40db-a656-73bea516fe41.png)
![image](https://user-images.githubusercontent.com/84165941/229016700-9f7d9e49-9892-4867-9577-6cbdfcd8281a.png)
![image](https://user-images.githubusercontent.com/84165941/229016723-f268dbd8-c885-4de0-98a8-aeadaa6d26a3.png)
![image](https://user-images.githubusercontent.com/84165941/229016747-816bb7db-8f82-4397-8610-d5dfd231605e.png)
![image](https://user-images.githubusercontent.com/84165941/229016779-a0a34226-ba46-48af-bb6a-a34653801932.png)

**Comment by namanpatel12** [comment]
Hi @NowinskiK ,

Just wonder if you can help on this as it's not working for me using minus sign. 

Would be really appreciated your help if you can direct.
